cpp/ps0001/problem solve 1.cpp

```diff
@@ -1,19 +1,23 @@
 #include <iostream>
 using namespace std;
 int main(){
-    int first;
-    int second;
-    int third;
-    int sum;
-    cout <<"please enter first number\n";
-    cin >>first;
-    cout <<"please enter second number\n";
-    cin >>second;
-    cout <<"please enter third number\n";
-    cin >>third;
+    // prompting user for 3 numbers to add 1/2
+    short first;
+    short second;
+    short third;
+    short sum;
+    cout << "please enter first number\n";
+    cin >> first;
+    cout << "please enter second number\n";
+    cin >> second;
+    cout << "please enter third number\n";
+    cin >> third;
+
+    // outputting the result of the summation process 3/3
     sum = (first + second + third);
-    cout <<"sum : \n";
+    cout << "sum : \n";
     cout << sum;

```

Comment:
- [x] did summation process for the three inputted numbers
- [x] used `std::cout` in the right way to display output from the summation process
- [ ] output is as required
```
155
940
168
--------
1263
```
+1 bonus mar as your program is user-friendly ⭐